src/util/postgres_client.js

```diff
@@ -742,10 +742,27 @@ class PostgresTable {
 
     async find(query, options = {}) {
 
+        function isObject(v) {
+            return (typeof v === 'object' && !Array.isArray(v) && v !== null);
+        }
+        function calculateOptionsAndArrayFields(q) {
+            const ops = [ '$all' ];
+            const l = [];
+            // eslint-disable-next-line guard-for-in
+            for (const p in q) {
```

Comment:
in general `for in` is discouraged, hence the eslint rule. see here https://eslint.org/docs/rules/guard-for-in

instead, you can use this:
```suggestion
            for (const key of Object.keys(q)) {
```

Reply:
👍